docs/kr.tree

```diff
@@ -282,6 +283,8 @@
     </toc-element>
 
     <toc-element toc-title="Standard library">
+
+        <toc-element id="read-standard-input.md" toc-title="Read standard input"/>
```

Comment:
Will it be compatible if we move the page later to the IO subsection when we have more material on that?

Reply:
Can you point me to where exactly you're thinking? I searched for IO content and what I found is in the API reference.

Reply:
I mean if it is now in "Standard library / Read standard input" and we later move this page to "Standard library / IO / Read standard input", it won't affect the page url?

Reply:
I'd also place it after collections

Reply:
Whenever we want to expand the content and have the IO section, we can do the proper updates and redirects.